app/src/renderer/components/staking/LiValidator.vue

```diff
@@ -141,50 +141,20 @@ export default {
 </script>
 
 <style>
-.li-validator {
-  padding: 1rem;
-  background-color: var(--app-fg);
-  border-radius: 0.25rem;
-  border: 1px solid var(--bc-dim);
-}
-.li-validator:hover {
-  background: var(--hover-bg);
-}
-.li-validator .validator-profile__status {
-  left: 0;
-  top: 9px;
-}
-.li-validator__name-container {
-  position: relative;
-  margin-left: 0.5rem;
-}
-.li-validator__name-container .li-validator__moniker {
-  padding-left: 0.75rem;
-}
-.li-validator__moniker-container {
-  display: flex;
-  align-items: center;
-  width: 100%;
-  min-width: 284px;
-}
-.li-validator__moniker {
-  max-width: 200px;
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-}
-.li-validator__avatar {
-  height: 3rem;
-  width: 3rem;
-  margin: 1rem 0.5rem;
-  border-radius: 50%;
+.li-validator.data-table__row td:nth-child(3):after {
```

Comment:
Really minor: I'm thinking if relying on the child order will be a strong assumption, maybe we can add a class to define the separation explicitely

Reply:
Refactored